image-viewer.py

```diff
@@ -1,35 +1,53 @@
+from operator import itemgetter
 from renderer import Renderer
 from menu import AppMenu
 from tkinter import Tk, Label, Button
 from PIL import Image, ImageTk
+import json
+
 
 class ImageViewer(Renderer, AppMenu):
     def __init__(self):
-        
+
+        f = open("./settings/default_display_settings.json", "r")
+        f2 = open("./lang/en-GB.json")
```

Comment:
I'd suggest renaming `f` to defaultDisplaySettingsFile and `f2` to `languageFile`, purely just for readability :)

Reply:
Ok, I agree. Done